sample/src/main/java/com/airbnb/airmapview/sample/MainActivity.java

```diff
@@ -86,6 +86,25 @@
     return super.onOptionsItemSelected(item);
   }
 
+  private AirMapInterface createMapWithTouchDisabled(AirMapViewTypes type) {
+    if (type == AirMapViewTypes.NATIVE) {
+      AirGoogleMapOptions options = new AirGoogleMapOptions(new GoogleMapOptions());
+      options.setTouchEnabled(false);
+      AirMapInterface airMapInterface = mapViewBuilder.builder(type).withOptions(options).build();
+      return airMapInterface;
+    }
+
+    // case where it's not a native map, doesn't use AirGoogleMapOptions
```

Comment:
@petzel I'm having a little bit of trouble with using the options for the NATIVE type -- it won't accept an instance of ```AirGoogleMapOptions``` (it takes an ```AirMapType```).

Therefore I proceeded to using the code below. Other the ```OnMapLoadedListener``` gets overridden by this activity's ```onMapInitialized``` method. Is there a way for me to overcome this? Thanks